html.go

```diff
@@ -20,7 +20,7 @@ func (req *HTMLRequest) Assets(assets ...Document) {
 }
 
 func (req *HTMLRequest) postURL() string {
-	return "/convert/html"
+	return "/forms/chromium/convert/html"
```

Comment:
Note: This will work with gotenberg API docker iamge v7, but is not backward compatible with v6.